packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -377,25 +383,27 @@ class InspectorTreeController extends Object
     root.children.forEach(_collapseAllNodes);
   }
 
-  int get numRows => root != null ? root.subtreeSize : 0;
+  int get numRows => root?.subtreeSize ?? 0;
 
   int getRowIndex(double y) => max(0, (y - verticalPadding) ~/ rowHeight);
 
-  InspectorTreeRow getRowForNode(InspectorTreeNode node) {
-    return getCachedRow(root.getRowIndex(node));
+  InspectorTreeRow? getRowForNode(InspectorTreeNode node) {
+    return getCachedRow(root!.getRowIndex(node));
```

Comment:
rather than using `root!`
return null early if root is null That is more coherent with the overall method that can return null.

Reply:
done